CONTRIBUTING.rst

```diff
@@ -21,3 +21,21 @@ ease-of-use for the user interface as well as naming suggestions for the
 various objects and options that users interact with.
 While stabilizing the interface will take precedence for the time being,
 new feature requests are always welcome!
+
+Development Details
+-------------------
+
+* Begin development work on ``sciform`` by forking and installing the
+  `git repository <https://github.com/jagerber48/sciform>`_ and
+  installing the development dependencies in editable mode with::
+
+     python -m pip install -e .[dev]
+* Tests can be run using::
+
+     python -m unittest
+* ``sciform`` is formatted using the
+  `ruff linter and formatter <https://docs.astral.sh/ruff/>`_.
+  Code should pass the following checks with no errors::
+
+     ruff check .
+     ruff format . --check
```

Comment:
When running this out of the box with a fresh ruff install, `ruff format . --check` gets this warning. I don't think you need to take any action, but might be something to file for future maintenance.

```bash
(pydemo) isabelzimmerman@Isabels-MBP-3 sciform % ruff format . --check
warning: The following rules may cause conflicts when used with the formatter: `COM812`, `ISC001`. To avoid unexpected behavior, we recommend disabling these rules, either by removing them from the `select` or `extend-select` configuration, or adding them to the `ignore` configuration.
27 files left unchanged
```

Reply:
Yes, I've seen that warning. Not sure how to clear it since I want to use rule COM812 in the linter and I also want to use the formatter. I opened an issue with Ruff to try to get guidance on this https://github.com/astral-sh/ruff/issues/9216.

I'd be fine to ignore rule ISC001 so maybe I'll do that.

I'll probably leave the config how it is for now and keep this warning.